hypothesis-python/setup.py

```diff
@@ -63,6 +63,7 @@ def local_file(name):
     "pytest": ["pytest>=4.6"],
     "dpcontracts": ["dpcontracts>=0.4"],
     "redis": ["redis>=3.0.0"],
+    "github-actions": ["requests >= 2.22.0"],
```

Comment:
If we used [the stdlib `urllib.request` module](https://docs.python.org/3/library/urllib.request.html) the implementation would be more verbose, but we'd avoid a dependency and could include this in the core `hypothesis.database` module.

Reply:
It's a pretty straightforward change. I'll do it after implementing the cache behaviour.